geometry_msgs/Point roi_0
geometry_msgs/Point roi_1
geometry_msgs/Point roi_2
geometry_msgs/Point roi_3
sensor_msgs/Image orig_image
